ci/scripts/misc-check.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Exits as soon as any line fails.
+set -euo pipefail
+
+echo "--- Install required tools"
```

Comment:
Install it in Dockerfile. Also I think for such trivial checks, we can still use GitHub Action.

Reply:
Yes, I think I can continue to use GitHub Action to run tests without ec2 and artifacts because there is no similar plugins in buildkite.